### PR TITLE
feat: Configure tools application registry category - EXO-61480

### PR DESCRIPTION
**Pull request by sofyenne**
This change is going to add the document portlet under the tools application category

### CODE REVIEW

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/application-registry-configuration.xml [hunk 1]
```diff
@@ -279,5 +279,55 @@
...
+                    <field name="applicationName"><string>DocumentsPortlet</string></field>
+                    <field name="categoryName"><string>tools</string></field>
+                    <field name="displayName"><string>Documents</string></field>
+                    <field name="description"><string>DocumentsPortlet</string></field>
+                    <field name="type"><string>portlet</string></field>
+                    <field name="contentId"><string>documents/DocumentsPortlet</string></field>
+                    <field name="accessPermissions">
+                      <collection type="java.util.ArrayList" item-type="java.lang.String">
+                        <value><string>*:/platform/users</string></value>
+                      </collection>
```
Comment (azayati) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/application-registry-configuration.xml]: separate the tags

**Review by azayati — CHANGES_REQUESTED** [review]

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/application-registry-configuration.xml (hunk 1 above)
Comment (sofyenne) [reply ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/application-registry-configuration.xml]: done !

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**